Sources/KituraContracts/Contracts.swift

```diff
@@ -310,6 +322,12 @@ public protocol Identifier {
 
 /**
  Extends `String` to comply to the `Identifier` protocol.
+ 
+### Usage Example: ###
+ ````swift
+ //The Identifier used in the Id field could be a `String`.
```

Comment:
Please can you put a space in between the // and the text for all of your examples as it makes it much easier to read. e.g. `// The Identifier used in the Id field could be a `String`.`